rasa/core/actions/action.py

```diff
@@ -253,12 +253,8 @@ def event_for_successful_execution(
         )
 
 
-class ActionUtterTemplate(Action):
-    """An action which only effect is to utter a template when it is run.
-
-    Both, name and utter template, need to be specified using
-    the `name` method.
-    """
+class ActionBotResponse(Action):
```

Comment:
This should also go in the changelog/migration guide if we are okay changing it right?

Reply:
Renaming the action class shouldn't affect anybody, no? In my opinion fine to change this without documenting it.